src/main.clj

```diff
@@ -201,6 +201,13 @@
                        )
                      ))))
              [new-m m-lvl [:error :fct :args-or-others]]))
+    :if (let [[condition is-true is-false] nc
+              evaluated-condition (last (new-eval m m-lvl condition))]
+          (if (zero? (:value evaluated-condition))
+            (new-eval m m-lvl is-true)
+            (if (= (first is-false) :else)
```

Comment:
What would be the conditions for `(first is-false)` to be something other than `:else` or `nil`?